### PR TITLE
Ks/proposed fixes create namespace for OpenPegasus and test

### PR DESCRIPTION
**Pull request by KSchopmeyer**
This is just a proposal for pegasus to allow the existing OpenPegasus to do create namespace as an extension to pr #1318.

It also includes the test I used in run_cimoperations.py to test the code.

### CODE REVIEW

File: pywbem/_server.py [hunk 1]
```diff
@@ -424,7 +428,7 @@ def create_namespace(self, namespace):
         ns_inst['CreationClassName'] = 'CIM_Namespace'
         ns_inst['Name'] = std_namespace
 
-        self.conn.CreateInstance(ns_inst, namespace=self.interop_ns)
+        path = self.conn.CreateInstance(ns_inst, namespace=self.interop_ns)
```
Comment (andy-maier) [inline pywbem/_server.py]: The `path` is not used and will fail the flake8 check.

File: testsuite/run_cim_operations.py [hunk 2]
```diff
@@ -4503,6 +4503,43 @@ def test_server_select_profiles(self):
         self.assertEqual(len(server.get_selected_profiles('blah', 'blah')), 0)
         self.assertEqual(len(server.get_selected_profiles('blah')), 0)
 
+    def test_create_namespace(self):
+        """Test the creat_namespace method."""
+
+        server = WBEMServer(self.conn)
+        namespaces = server.namespaces
+        interop = server.interop_ns
+        test_ns = "root/runcimoperationstestns"
```
Comment (andy-maier) [inline testsuite/run_cim_operations.py]: I think given that root has this special role, we should also test creation of namespaces that do not begin with `root/`.

Comment (KSchopmeyer) [reply testsuite/run_cim_operations.py]: Agreed.  In fact I want to add code that creates a qualifier decl to really establish that there is a working namespace.

The failure is in the tests, in particular I used PG_Interop and that is not mocked.

**Comment by andy-maier** [comment]
The remaining errors are:
* `path`variable assigned but not used.
* The newly used "PG_Namespace" class is not in the mocked repository that is used by the test_wbemserverclass.py test cases. It should be added in the build_class_repo() method in wbemserver_mock.py.

**Comment by andy-maier** [comment]
Karl, I have updated my PR #1318 to include the changes to the `pywbem/_server.py` file. It should now be complete w.r.t. the new create_namespace() method. The deletion method is left for a future PR.

You can now update this PR here to provide just the changes to the `run_cim_operations.py` file on top of my PR.

**Comment by KSchopmeyer** [comment]
This one had gotten so confusing that I am closing it and creating a new one to incorporate the run_cimoperation changes.

**Pull request closed without merging**